setup.py

```diff
@@ -3,40 +3,44 @@
 from setuptools import setup
```

Comment:
we really should move to a "modern" build -- e.g. not putting code in setup.py.

maybe even scikit build?